.github/workflows/gh_pages.yml

```diff
@@ -22,10 +22,11 @@ jobs:
         run: gulp
 
       - name: Checkout GH pages
-        run: |
-          git config user.name github-actions
-          git config user.email github-actions@github.com
-          git clone -b gh-pages "git@github.com:stellar/js-stellar-base.git" jsdoc
+        uses: actions/checkout@v2
+        with:
+          repository: stellar/js-stellar-base
+          ref: gh-pages
+          path: jsdoc
 
       - name: Generate JS docs
         run: yarn docs
```

Comment:
I'm not sure if you'll be at root `/` or at `/jsdoc` when you run the yarn docs command 🤔 . Let's merge it to find out 😄